src/Orckestra.Composer.Cart/Api/CartController.cs

```diff
@@ -144,7 +144,7 @@ public virtual async Task<IHttpActionResult> GetShippingMethods()
                 CultureInfo = ComposerContext.CultureInfo,
                 CustomerId = ComposerContext.CustomerId,
                 CartName = CartConfiguration.ShoppingCartName
-            });
```

Comment:
Do you check another places? For example line 64 without ConfigureAwait?

Reply:
@neexite updated controller